monero-outreach-docs/translations/pt/PSA/files/forks/forks_pt.md

```diff
@@ -0,0 +1,28 @@
+# Conheça os Hard Forks
+
+A menos que você seja um programador, o termo "hard fork" pode ter muitas conotações confusas e potencialmente negativas para si. Você poderá ter lido um artigo ou dois sobre "hard forks" e como isso poderá ter criado tensão e algum drama entre outros projetos de criptografia. Os "hard forks" de Monero podem classificar-se em três categorias principais:
+
+### 1) Atualização da Rede/"Hard Fork" Periódico
+
+Os programadores do Monero visam programar dois "hard fork" regularmente por ano. Esses "hard forks" são sempre feitos com o consenso da comunidade e liderados pelos programadores do núcleo do Monero. Normalmente incluem várias melhorias no código principal do Monero. Como utilizador do Monero é fundamental, que você atualize a sua carteira do Monero toda vez que ocorra um "hard fork". Ser um pioneiro não é fácil, mas se estiver atento às datas em que irão ocorrer estes "hard-fork" não será difícil também manter a sua carteira atualizada.
+
+Outros projetos usaram os "hard fork" regularmente programados do Monero em benefício próprio e criaram os seus próprios projetos nas antigas redes do Monero. Esses projetos normalmente falham em pouco tempo e  alimentam-se da ganância das pessoas. Se você não atualizar a sua carteira do Monero após um "hard fork" e enviar o seu XMR, poderá acabar perdendo-os na rede antiga ou ter que perder muito tempo para tentar recuperá-los.
+
+_Safety level:_ Best practice is to upgrade your Monero wallet(s) as soon as updates are released.
```

Comment:
My suggestion as a Brazilian:

`_Nível de segurança:_ Melhores práticas é atualizar a sua carteira do Monero assim que atualizações estiverem disponíveis.`

Reply:
Nice. Just a reminder about the PDF, it wasn't translated because the other text was occupying most of the space of the paragraph. I didn't find it by just looking at it in Scribbus, it was hidden since it is in another line but the line is not visible in the SLA nor in the PDF. More graphic:

https://imgur.com/a/4Y1gkfx